Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -332,26 +332,29 @@ using FullTrack = FullTracks::iterator;
-DECLARE_SOA_COLUMN(MatchMCHTrackID, matchMCHTrackID, int);     //! ID of matching MCH track for GlobalMuonTracks  (ints while self indexing not available)
-DECLARE_SOA_COLUMN(MCHBitMap, MchBitMap, uint16_t);            //! Fired muon trackig chambers bitmap
-DECLARE_SOA_DYNAMIC_COLUMN(Sign, sign, //!
+DECLARE_SOA_SLICE_INDEX_COLUMN(BC, bc); //! BC index (slice for 1 to N entries)
+// TODO to be replaced by a variable length array
+DECLARE_SOA_ARRAY_INDEX_COLUMN(Collision, collision, 2); //! Array of Collision IDs for forward tracks
+DECLARE_SOA_INDEX_COLUMN(Collision, collision);          //! Collision ID for mft tracks
+// TODO change to ForwardTrackTypeEnum when enums are supported
+DECLARE_SOA_COLUMN(TrackType, trackType, uint8_t);                                        //! Type of track. See enum ForwardTrackTypeEnum
+DECLARE_SOA_COLUMN(X, x, float);                                                          //! TrackParFwd parameter x
+DECLARE_SOA_COLUMN(Y, y, float);                                                          //! TrackParFwd parameter y
+DECLARE_SOA_COLUMN(Z, z, float);                                                          //! TrackParFwd propagation parameter z
+DECLARE_SOA_COLUMN(Phi, phi, float);                                                      //! TrackParFwd parameter phi; (i.e. pt pointing direction)
+DECLARE_SOA_COLUMN(Tgl, tgl, float);                                                      //! TrackParFwd parameter tan(\lamba); (\lambda = 90 - \theta_{polar})
+DECLARE_SOA_COLUMN(Signed1Pt, signed1Pt, float);                                          //! TrackParFwd parameter: charged inverse transverse momentum; (q/pt)
+DECLARE_SOA_COLUMN(NClusters, nClusters, int8_t);                                         //! Number of clusters
+DECLARE_SOA_COLUMN(Chi2, chi2, float);                                                    //! Track chi^2
+DECLARE_SOA_COLUMN(PDca, pDca, float);                                                    //! PDca for MUONStandalone
+DECLARE_SOA_COLUMN(RAtAbsorberEnd, rAtAbsorberEnd, float);                                //! RAtAbsorberEnd for MUONStandalone tracks and GlobalMuonTrackstracks
+DECLARE_SOA_COLUMN(Chi2MatchMCHMID, chi2MatchMCHMID, float);                              //! MCH-MID Match Chi2 for MUONStandalone tracks
+DECLARE_SOA_COLUMN(Chi2MatchMCHMFT, chi2MatchMCHMFT, float);                              //! MCH-MFT Match Chi2 for GlobalMuonTracks
+DECLARE_SOA_COLUMN(MatchScoreMCHMFT, matchScoreMCHMFT, float);                            //! MCH-MFT Machine Learning Matching Score for GlobalMuonTracks
+DECLARE_SOA_COLUMN(MatchMFTTrackID, matchMFTTrackID, int);                                //! ID of matching MFT track for GlobalMuonTracks
+DECLARE_SOA_SELF_INDEX_COLUMN_FULL(MatchMCHTrack, matchMCHTrack, int, "MatchMCHTrackID"); //! ID of matching MCH track for GlobalMuonTracks
```

Comment:
In the file this should be the column `fIndexFwdTracks_MatchMCHTrackID`
You can probably reduce MatchMCHTrackID. At least ID can be removed.